stdlib/universe/universe.flux

```diff
@@ -214,6 +214,11 @@ builtin logarithmicBins : (start: float, factor: float, count: int, ?infinity: b
 // die returns a fatal error from within a flux script
 builtin die : (msg: string) => A
 
+// Returns true if `v` is an object matching `type`.
+// This function can only check against the primitive types:
+// `string`, `bytes`, `int`, `uint`, `float`, `bool`, `time`, `duration`, `regexp`.
+builtin isType : (v: A, type: string) => bool
```

Comment:
I like the simplicity of this function. Any reason we would put this into its own package? Generally universe is quite large so I'd prefer we don't add to it but happy to be convinced otherwise.

Reply:
I just put it here for simplicity, I don't have an opinion or context towards where it would make the most sense to put this :confused: